cmd/crictl/sandbox.go

```diff
@@ -103,16 +103,33 @@ var removePodCommand = cli.Command{
 	Name:      "rmp",
 	Usage:     "Remove one or more pods",
 	ArgsUsage: "POD-ID [POD-ID...]",
-	Action: func(context *cli.Context) error {
-		if context.NArg() == 0 {
-			return cli.ShowSubcommandHelp(context)
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "Force removal of the pod sandbox, disregarding if running",
+		},
+	},
+	Action: func(ctx *cli.Context) error {
+		if ctx.NArg() == 0 {
+			return cli.ShowSubcommandHelp(ctx)
 		}
-		if err := getRuntimeClient(context); err != nil {
+		if err := getRuntimeClient(ctx); err != nil {
 			return err
 		}
-		for i := 0; i < context.NArg(); i++ {
-			id := context.Args().Get(i)
-			err := RemovePodSandbox(runtimeClient, id)
+		for i := 0; i < ctx.NArg(); i++ {
+			id := ctx.Args().Get(i)
+
+			resp, err := runtimeClient.PodSandboxStatus(context.Background(),
+				&pb.PodSandboxStatusRequest{PodSandboxId: id})
+			if err != nil {
+				return err
+			}
+			if !ctx.Bool("force") &&
```

Comment:
if force is true and sandbox state is READY, then invoke `StopPodSandbox()` before `RemovePodSandbox`?

Reply:
Yes that sounds good, I will change it as adapted.